src/openforms/js/lang/nl.json

```diff
@@ -1295,7 +1320,7 @@
     "originalDefault": "Object"
   },
   "fpX4ei": {
-    "defaultMessage": "Mapping expression language",
+    "defaultMessage": "Soort mappingexpressie",
```

Comment:
Not sure but use "taal" instead of "soort"?

Reply:
I don't think we want users to really think about "programming languages" or get confused with language as in English vs. Dutch - ultimately the feature is about configuring an expression to extract data, which happens to be implemented in a "language". The important information however is designating what kind of expression it is (the original source string is not ideal either in that regard).